### ISSUE TITLE
Error with composer update on windows

### ISSUE THREAD

**Issue by temp** [puli/puli#22]
A colleague just checked out my puli-branch and did a composer update, on a windows machine.

It produced this error while calling "composer update", when trying to build the puli repository:

```
[ErrorException]  
Uninitialized string offset: 0

Exception trace:
() at C:\Users\xxx\Projects\myproject\vendor\webmozart\path-util\src\Path.php:567
Composer\Util\ErrorHandler::handle() at C:\Users\xxx\Projects\myproject\vendor\webmozart\path-util\src\Path.php:567
Webmozart\PathUtil\Path::split() at C:\Users\xxx\Projects\myproject\vendor\webmozart\path-util\src\Path.php:355
Webmozart\PathUtil\Path::makeRelative() at C:\Users\xxx\Projects\myproject\vendor\puli\composer-plugin\src\PuliPlugin.php:116
Puli\Extension\Composer\PuliPlugin->installNewPackages() at C:\Users\xxx\Projects\myproject\vendor\puli\composer-plugin\src\PuliPlugin.php:85
Puli\Extension\Composer\PuliPlugin->postInstall() at n/a:n/a                                                                                                                                                                                   
call_user_func() at phar://C:/ProgramData/ComposerSetup/bin/composer.phar/src/Composer/EventDispatcher/EventDispatcher.php:158
Composer\EventDispatcher\EventDispatcher->doDispatch() at phar://C:/ProgramData/ComposerSetup/bin/composer.phar/src/Composer/EventDispatcher/EventDispatcher.php:120
Composer\EventDispatcher\EventDispatcher->dispatchCommandEvent() at phar://C:/ProgramData/ComposerSetup/bin/composer.phar/src/Composer/Installer.php:330Composer\Installer->run() at phar://C:/ProgramData/ComposerSetup/bin/composer.phar/src/Composer/Command/InstallCommand.php:125
Composer\Command\InstallCommand->execute() at phar://C:/ProgramData/ComposerSetup/bin/composer.phar/vendor/symfony/console/Symfony/Component/Console/Command/Command.php:252
Symfony\Component\Console\Command\Command->run() at phar://C:/ProgramData/ComposerSetup/bin/composer.phar/vendor/symfony/console/Symfony/Component/Console/Application.php:874
Symfony\Component\Console\Application->doRunCommand() at phar://C:/ProgramData/ComposerSetup/bin/composer.phar/vendor/symfony/console/Symfony/Component/Console/Application.php:195
Symfony\Component\Console\Application->doRun() at phar://C:/ProgramData/ComposerSetup/bin/composer.phar/src/Composer/Console/Application.php:146Composer\Console\Application->doRun() at phar://C:/ProgramData/ComposerSetup/bin/composer.phar/vendor/symfony/console/Symfony/Component/Console/Application.php:126
Symfony\Component\Console\Application->run() at phar://C:/ProgramData/ComposerSetup/bin/composer.phar/src/Composer/Console/Application.php:83                                                                                                  
Composer\Console\Application->run() at phar://C:/ProgramData/ComposerSetup/bin/composer.phar/bin/composer:43
require() at C:\ProgramData\ComposerSetup\bin\composer.phar:24
```

"bin/puli build" worked fine, nevertheless.

**Comment by webmozart** [comment]
Thanks for the report! The package synchronization with Composer is a little flawed at the moment. I'm going to fix that ASAP.

**Comment by temp** [comment]
Thanks! Let me know if you need more input.

**Comment by webmozart** [comment]
At PuliPlugin:116, what are the values of the variables passed to `Path::makeRelative()`?

**Comment by temp** [comment]
Here we go, path is anonymized, installPath points to project root dir:
```
$installPath: C:\Users\xxx\Projects\myproject
$rootDir: (empty string)
```

**Comment by webmozart** [comment]
Thanks! This should be fixed by webmozart/path-util@abe2a6cba8ecd16e834ef09472719c448382b905, could you check?

**Comment by temp** [comment]
Works, thanks!
